Remove redis lib/dep

camilamacedo86 opened the pull request:
# Jira link(s)

https://issues.jboss.org/browse/RHMAP-21576

# What
Remove the redis lib

# Why
It is unused.

# How
`npm uninstall --save redis`

# Verification Steps
The usage of this lib/dep is just for the test [here](https://github.com/feedhenry/fh-mbaas-middleware/blob/7872b4b67a2c3abc96de0e8c2378fb215ec23c20/test/fixtures/monitorcommon.js#L69) then check if the CI finish with success should be enough. 

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

Commit:
Remove unused Redis dep

Commit:
Bump version

grdryn commented:
@camilamacedo86 I don't think the `redis` module is actually used in this project, is it? If not, then maybe we should just remove it, rather than updating it?

camilamacedo86 commented:
@grdryn the redis is required in the tests. See [here](https://github.com/feedhenry/fh-mbaas-middleware/blob/7872b4b67a2c3abc96de0e8c2378fb215ec23c20/test/fixtures/monitorcommon.js#L14)

grdryn commented:
@camilamacedo86 Is it though? That looks to be just assigning a function to a variable named `redis` (as a test mock/stub), rather than using the `redis` dependency. Does that actually cause the dependency module to be loaded?

camilamacedo86 commented:
@grdryn you are 100% correct. Thank your for see it. 

I used the deep check and it was defined as unused lib too. I removed it and updated the first comment. Please, feel free to review.
